datamechanics_airflow_plugin/operator.py

```diff
@@ -45,22 +54,18 @@ def __init__(
         self.dm_retry_limit = dm_retry_limit
         self.dm_retry_delay = dm_retry_delay
         self.app_name = None  # will be set from the API response
-
+        self._paylod_app_name = app_name
```

Comment:
nit:
```suggestion
        self._payload_app_name = app_name
```

Reply:
🤦  At least I was consistent! Pushing an update to correct the other references.